build/chip/chip_codegen.cmake

```diff
@@ -135,11 +136,13 @@ endfunction()
 #
 #   OUTPUT_FILES - [OUT] output variable will contain the path of generated files.
 #                  suitable to be added within a build target
+# ZCL_PATH      - [OPTIONAL] path to a custom ZCL JSON file. If provided, it overrides the default ZCL path used by generate.py.
```

Comment:
please make this to be in line with previous docs: alignment of both the argument and then the text is indented on new lines